docs/contributor/developing.md

```diff
@@ -75,19 +75,35 @@
 
 1. Run a Basic Syntax/Lint Check (using flake8) and resolve any issues
 
-    ``` cmd
```

Comment:
This is out of date. pydocstyle and flake8 were replaced with `ruff`. The new command that should be documented is `ruff edk2toollib` and `ruff` comes installed with the command `pip install -e .[dev]`

Reply:
Updated to mention the pip install command earlier in the file and replace `flake8` with `ruff`.